pycket/entry_point.py

```diff
@@ -44,9 +44,20 @@ def actual_entry(argv):
         modtable = ModTable()
         modtable.enter_module(module_name)
         if json_ast is None:
-            ast = expand_to_ast(module_name, modtable)
+            if 'byteRecursive' in names:
```

Comment:
Can we hide all this logic in the expand module where all the other expansion logic exists? Perhaps just have `expand_to_ast` take a boolean or enum argument to specify whether or not to use bytecode expansion.